mlflow/utils/gorilla.py

```diff
@@ -292,10 +293,20 @@ def apply(patch):
     """
     settings = Settings() if patch.settings is None else patch.settings
 
+    curr_active_patch = _ACTIVE_PATCH % (patch.name,)
+    if curr_active_patch in patch.destination.__dict__:
+        # safe guarding checking.
+        raise RuntimeError(
+            f"Patch {patch.name} on {destination.__name__} already existed."
+            f"Revert old patch first if you want to patch it again."
+        )
```

Comment:
Fast fail if it already patched. Fast fail will help debug and prevent unpredictable bug.

Reply:
I change this to a warning. raise error here breaks many tests.